src/controllers/referral.js

```diff
@@ -20,10 +20,11 @@ const {
   COMPLETED_SUCCESSFULLY,
 } = messages;
 
-const getSubjectUpi = async (echisClientId) => {
+const getSubjectUpi = async (echisClientId,instance) => {
```

Comment:
We can rearrange the order of params here to match other functions that expect and instance object

Reply:
Should we have it come first?

Reply:
Updated the functions.